### PR TITLE
Added packagehub related test modules to extra_tests_textmode_phub job

### PR DESCRIPTION
**Pull request by tinawang123**
Add libgit2 test module to extra_tests_textmode_phub job

- Related ticket: https://progress.opensuse.org/issues/167419
- Verification run: https://openqa.suse.de/tests/15546984#
    https://openqa.suse.de/tests/15547383
    https://openqa.suse.de/tests/15546996#

### CODE REVIEW

File: schedule/functional/extra_tests_textmode_phub.yaml [hunk 1]
```diff
@@ -9,7 +9,9 @@ schedule:
     - '{{wpa_supplicant}}'
     - console/vmstat
     - console/libjpeg_turbo
+    - console/libgit2
     - console/ansible
+    - network/cifs
```
Comment (rfan1) [inline schedule/functional/extra_tests_textmode_phub.yaml:14]: Is it the same reason as libgit2?

Comment (tinawang123) [reply schedule/functional/extra_tests_textmode_phub.yaml:14]: Yes, cifs need to install 'samba-client' which requires PackageHub

Comment (rfan1) [reply schedule/functional/extra_tests_textmode_phub.yaml:14]: Then I would suggest changing the git messages and title :)

Comment (tinawang123) [reply schedule/functional/extra_tests_textmode_phub.yaml:14]: Also this failed job: https://openqa.suse.de/tests/15534070#step/prepare_systemd_and_testsuite/56
I moved the 'prepare_systemd_and_testsuite' to extra_tests_textmode_phub job

**Review by foursixnine — APPROVED** [review]

**Comment by rfan1** [comment]
Seems a bit fast to merge this PR https://openqa.opensuse.org/tests/4513593, we may lose coverage on o3? @tinawang123 can you please create a new PR then?

**Comment by tinawang123** [comment]
> Seems a bit fast to merge this PR https://openqa.opensuse.org/tests/4513593, we may lose coverage on o3? @tinawang123 can you please create a new PR then?

o3 job doesn't need this fix. As it can use those packages without packagehub.

**Comment by rfan1** [comment]
> > Seems a bit fast to merge this PR https://openqa.opensuse.org/tests/4513593, we may lose coverage on o3? @tinawang123 can you please create a new PR then?
> 
> o3 job doesn't need this fix. As it can use those packages without packagehub.

But now `libgit2` test is removed from o3, we need to add it back :)

**Pull request merged**